### DISCUSSION TITLE
【委托】假期即将结束但某高中生作业一字未动该如何解决

### DISCUSSION THREAD

**Discussion by LiHongRS** [share121/inter-knot#982] [其他]
![008l6ZJTly1hraylya2gdj30rl0rl0ze](https://github.com/user-attachments/assets/fa1b2053-7c64-4f86-992a-2b8188270030)

如若有人能给与解决方案本人将给出10w丁尼的委托费

**Comment by Wzyaly** [comment]
什么学科的，多的话再加点，这年头答案可不好弄

**Comment by Yizakl** [comment]
换成10w菲林罢

**Reply by 10kaihua** [reply]
10w菲林你让我生吃以骸都可以🤩

**Comment by c1iied** [comment]
实在没办法就求同学
大家都没做就不用管😎

**Comment by Pessimi-AFK** [comment]
丢空洞，告诉老师作业掉空洞了（doge

**Comment by 290678** [comment]
告诉老师作业变成了未知复合侵蚀体（bushi）

**Comment by Yizakl** [comment]
去找闪老师
